optaplanner-core/src/benchmark/assembly/benchmarks.xml

```diff
@@ -0,0 +1,27 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+  <id>benchmarks</id>
+  <formats>
+    <format>jar</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <dependencySets>
+    <dependencySet>
+      <outputDirectory>/</outputDirectory>
```

Comment:
Can we replace `/` with `${file.separator}` (or `.` or empty value or remove `<outputDirectory>` completely) to avoid the following warning?
```
[INFO] --- maven-assembly-plugin:3.1.0:single (make-assembly) @ optaplanner-core ---
[INFO] Reading assembly descriptor: src/benchmark/assembly/benchmarks.xml
[WARNING] The assembly descriptor contains a filesystem-root relative reference, which is not cross platform compatible /
```
https://stackoverflow.com/questions/28500401/maven-assembly-plugin-warning-the-assembly-descriptor-contains-a-filesystem-roo/

Reply:
Nice catches! Fixed!